fud/fud/stages/xilinx/xclbin.py

```diff
@@ -108,7 +108,7 @@ def package_xo(client: SourceType.UnTyped, tmpdir: SourceType.String):
                 "/scratch/opt/Xilinx/Vivado/2020.2/bin/vivado "
                 "-mode batch "
                 "-source gen_xo.tcl "
-                f"-tclargs xclbin/kernel.xo"
+                f"-tclargs xclbin/kernel.xo 2"
```

Comment:
To make this work like it did before (and keep working on the tests we have set up), I think we want this to be 1?

Reply:
My proposal in https://github.com/cucapra/calyx/issues/853#issuecomment-1023470983 is to start with “NFC,” i.e., keeping the hard-coded behavior. Then a subsequent PR can make this actually match the program. My proposal there is something like parsing the XML to figure this out so the user doesn’t have to specify it.

Reply:
Seems reasonable!

Reply:
Yeah, the reason why I used numbers was simply to make the Tcl implementation simpler.

> My proposal in [#853 (comment)](https://github.com/cucapra/calyx/issues/853#issuecomment-1023470983) is to start with “NFC,” i.e., keeping the hard-coded behavior. Then a subsequent PR can make this actually match the program. My proposal there is something like parsing the XML to figure this out so the user doesn’t have to specify it.

That makes sense. Would parsing the XML be another step in `xclbin`?

Reply:
Yup, it can be its own stage that gets added later on.